src/main/resources/i18n/messages.properties

```diff
@@ -98,6 +98,9 @@ FFmpeg.0=<html>Font setting is applied to embedded and external text-based subti
 FFmpeg.1=Defer to MEncoder when transcoding problematic subtitles
 FFmpeg.2=<html>MEncoder is more stable than FFmpeg at transcoding some kinds of subtitles, so this lets us choose which is the most stable option for each video.</html>
 FFmpeg.3=Use font settings (size, color ...)
+FFmpeg.4=Output silent audio
+FFmpeg.5=<html> Output silent audio when a movie do not have audio in ffmpegvideo.<br>Some of renderers do not support a movie without audio.<br>This can be perfomed by either UMS.conf or renderer.conf, also.
+</html>
```

Comment:
This should probably be on the previous line, and do you mind if I do a commit to alter the English in here?

Reply:
No problem. I will correct only spaces.

Reply:
About " Are there any renderers in particular that this can be applied to straight away?"
Yes.

Reply:
@onon765trb great, let's add this for at least one renderer in this Pull Request, because it would be good for the feature to have at least one use

Reply:
@ SubJunk Does it mean that the changes of PmsConfiguration.java and GUI had better been deleted? I don't mind even if so. In so case, which will I or you do that?

Reply:
@SubJunk Nice! Do you commit it? I don't mind it. Do you think I had better add the feature to my other PRs, #1548 and #1549.

Reply:
@onon765trb I don't know which renderers do not support movies with no audio track, so I can't do it